chia/full_node/fee_tracker.py

```diff
@@ -0,0 +1,577 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import List, Optional, Tuple
+
+from sortedcontainers import SortedDict
+
+from chia.full_node.fee_estimate_store import FeeStore
+from chia.full_node.fee_estimator_constants import (
+    FEE_ESTIMATOR_VERSION,
+    INFINITE_FEE_RATE,
+    INITIAL_STEP,
+    LONG_BLOCK_PERIOD,
+    LONG_DECAY,
+    LONG_SCALE,
+    MAX_FEE_RATE,
+    MED_BLOCK_PERIOD,
+    MED_DECAY,
+    MED_SCALE,
+    SECONDS_PER_BLOCK,
+    SHORT_BLOCK_PERIOD,
+    SHORT_DECAY,
+    SHORT_SCALE,
+    STEP_SIZE,
+    SUCCESS_PCT,
+    SUFFICIENT_FEE_TXS,
+)
+from chia.full_node.fee_history import FeeStatBackup, FeeTrackerBackup
+from chia.types.mempool_item import MempoolItem
+from chia.util.ints import uint8, uint32, uint64
+
+
+@dataclass
+class BucketResult:
+    start: float
+    end: float
+    within_target: float
+    total_confirmed: float
+    in_mempool: float
+    left_mempool: float
+
+
+@dataclass
+class EstimateResult:
+    requested_time: uint64
+    pass_bucket: BucketResult
+    fail_bucket: BucketResult
+    median: float
+
+
+def get_estimate_block_intervals() -> List[int]:
+    return [
+        SHORT_BLOCK_PERIOD * SHORT_SCALE - SHORT_SCALE,
```

Comment:
why do wee need both a period and a scale for each interval  ?

im trying to better understand this calculation